docs/autolinking.md

```diff
@@ -1,20 +1,14 @@
 # Autolinking
 
-Autolinking is a mechanism built into CLI that allows adding a dependency with native components for React Native to be as simple as:
+React Native libraries usually come with platform-specific (native) code. Autolinking is a mechanism that allows your project to discover and use this code.
```

Comment:
"often" instead of "usually"

Reply:
Ah, that was my first attempt and I changed it :P